src/webhooks/registry.ts

```diff
@@ -256,7 +259,9 @@ const WebhooksRegistry: RegistryInterface = {
   },
 
   getHandler(topic: string): WebhookRegistryEntry | null {
-    return topic in WebhooksRegistry.webhookRegistry ? WebhooksRegistry.webhookRegistry[topic] : null;
+    return topic in WebhooksRegistry.webhookRegistry
+      ? WebhooksRegistry.webhookRegistry[topic]
+      : null;
```

Comment:
I believe we're doing this to avoid returning an `undefined` - would this simplify the code?
```suggestion
    return WebhooksRegistry.webhookRegistry[topic] ?? null;
```